simplewebrtc.js

```diff
@@ -60,11 +60,15 @@ function SimpleWebRTC(opts) {
     // call WildEmitter constructor
     WildEmitter.call(this);
 
-    // our socket.io connection
-    connection = this.connection = io.connect(this.config.url, this.config.socketio);
+    // create default SocketIoConnection if it's not passed in
+    if (this.config.connection === null) {
+        connection = this.connection = new SocketIoConnection(opts);
```

Comment:
:+1: I didn't test it. I was thinking this is a proof of concept :stuck_out_tongue_winking_eye: